### PR TITLE
Generate and publish nuget ab#130932

### PR DESCRIPTION
**Pull request by DfernandezNipo**
ab#130932

Generate and publish nuget.

https://github.com/NIPOSoftwareBV/Nfield-SDK/pull/436
https://github.com/NIPOSoftwareBV/Nfield.Quota/pull/116

### CODE REVIEW

File: pipelines/deploy-nuget.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: $(SourceBranchName)-$(date:yyyyMMdd)$(rev:.r)
+
+pr: none
+
+trigger: none
+
+# This pipeline is triggered when the Build stage in the DomainApiBuild pipeline finishes
```
Comment (hcsauniere) [inline pipelines/deploy-nuget.yml:7]:
```suggestion
# This pipeline is triggered when the Build stage in the QuotaBuild pipeline finishes
```

File: devops.yml [hunk 2]
```diff
@@ -1,89 +0,0 @@
...
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish Artifact: drop'
-    inputs:
-      PathtoPublish: '$(build.artifactstagingdirectory)'
-  
-  # Create NuGet Version    
-  - task: PowerShell@1
-    displayName: 'Create NuGet Version'
-    inputs:
-      scriptName: createnugetversion.ps1
```
Comment (hcsauniere) [inline devops.yml:59]: Is "createnugetversion.ps1" or "publish-release.ps1" still used?

File: pipelines/YmlTemplates/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                      Write-Host "##vso[task.setvariable variable=BranchName]$BranchName"
+                      Write-Host "##vso[task.setvariable variable=ReleaseName]$ReleaseName"
+                      Write-Host "##vso[task.setvariable variable=CommitHash]$CommitHash"
+                      Write-Host "##vso[task.setvariable variable=VersionName]$VersionName"
+                      
+                - task: GitHubRelease@1
+                  displayName: 'Create GitHub Release'
+                  condition: ${{ eq(parameters.nuGetFeedType, 'external') }}
+                  inputs:
+                    gitHubConnection: 'github.com_AlbertoBelver' # OAuth Service Conection https://dev.azure.com/niposoftware/Nfield/_settings/adminservices
```
Comment (hcsauniere) [inline pipelines/YmlTemplates/deploy.yml:94]: This is a public repo. I really don't think that we should write here something like this

**Review by hcsauniere — COMMENTED** [review]
I haven't finished to review it, but it would help to have some notes on what are actually the objective of the changes. The package was already generating and publishing nuget packages no?

**Comment by DfernandezNipo** [comment]
> I haven't finished to review it, but it would help to have some notes on what are actually the objective of the changes. The package was already generating and publishing nuget packages no?

The build creates a nuget and publish it in the internal feed. Release publish it in the external feed (nuget)

**Review by AlbertoBelver — APPROVED** [review]

**Pull request merged**